pydocs/conf.py

```diff
@@ -0,0 +1,196 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('../python/src/exactextract'))
+
+# -- Project information -----------------------------------------------------
+
+project = 'exactextract'
+copyright = '2022, ISciences, LLC'
+author = 'ISciences, LLC'
+
+# The short X.Y version
+version = ''
```

Comment:
Definitely like this idea. Ideally we'd match any versioning that the main library does, but since that's not happening right now perhaps starting with something like 0.0.1 should suffice.